mlflow/store/tracking/rest_store.py

```diff
@@ -222,6 +223,7 @@ def log_param(self, run_id, param):
         :param run_id: String id for the run
         :param param: Param instance to log
         """
+        _validate_param(param.key, param.value)
```

Comment:
```suggestion
```

The rest store should just pass the given parameter to the backend server without validation. For example, you may want to implement your own backend that accepts a param value longer than 500.

Reply:
thanks, will fix

Reply:
will undo the same change also for `log_batch` in the same module